packages/desktop-client/src/components/filters/FiltersMenu.js

```diff
@@ -181,7 +181,8 @@ function ConfigureField({
         <View style={{ marginBottom: 10 }}>
           <Stack direction="row" align="flex-start">
             {field === 'amount' || field === 'date' ? (
-              <Select
+              <Select style={{ borderWidth: 1, borderColor: 'transparent' }}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think you can just remove the `style` proper entirely. That should solve the issue.

Reply:
Thank you for your feedback, you mean the style property of the View component?